pages/api/admin/users/[id]/index.js

```diff
@@ -0,0 +1,68 @@
+import { handleRequests } from "@klaudsol/commons/lib/API";
+import { withSession } from "@klaudsol/commons/lib/Session";
+import { createHash } from '@/lib/Hash';
+import { OK, NOT_FOUND } from '@klaudsol/commons/lib/HttpStatuses';
+import InsufficientDataError from '@klaudsol/commons/errors/InsufficientDataError';
+import UserAlreadyExists from "@klaudsol/commons/errors/UserAlreadyExists";
+import People from '@klaudsol/commons/models/People';
+import Groups from '@klaudsol/commons/models/Groups';
+import PeopleGroups from '@klaudsol/commons/models/PeopleGroups';
+
+export default withSession(handleRequests({ get, put, patch, del }));
+
+async function get(req, res) {
+    const { id } = req.query;
+    const person = await People.get({ id });
+    const groups = await Groups.findByUser({ id });
+
+    const output = {
+        data: { person, groups },
+        metadata: {},
+    };
+
+    output.metadata.hash = createHash(output);
+
+    person ? res.status(OK).json(output ?? []) : res.status(NOT_FOUND).json({});
+}
+
+async function put(req, res) {
+    const { id } = req.query; 
```

Comment:
Kindly use assertUserCan at the beginning of all API endpoints.

https://github.com/klaudsol/klaudsol-commons/blob/main/src/lib/Permissions.js#L78